gpflow/kernels/base.py

```diff
@@ -158,6 +158,9 @@ def __init__(self, kernels, name=None):
             raise TypeError(
                 "can only combine Kernel instances")  # pragma: no cover
 
+        self._set_kernels(kernels)
+
+    def _set_kernels(self, kernels):
```

Comment:
```suggestion
    def _set_kernels(self, kernels: List[Kernel]):
```

^^^ might require import List, or you can use Iteratable (if it allows to specify the element type)